docker-compose.yml

```diff
@@ -3,20 +3,24 @@ version: '3'
 services:
 
   database:
-    image: postgres:10.4
+    image: postgres:14
     container_name: database
     env_file:
       - .env.dev
     ports:
       - "5432:5432"
 
   api:
-    image: api
```

Comment:
Do we need to remove this line?

Reply:
We don't need to remove:
- It is from my personal setup of `docker-compose.yml`.
- Useful for getting instant updates from the `api`.
- I will remove it, since it would effect other developers builds.